config/fund_loader_config/digital_planning/dpi_r2.py

```diff
@@ -0,0 +1,149 @@
+from datetime import datetime
+from datetime import timezone
+
+from config.fund_loader_config.common_fund_config.fund_base_tree_paths import (
+    DPI_R2_BASE_PATH,
+)
+from config.fund_loader_config.logo import DLUHC_LOGO_PNG
+
+DPI_FUND_ID = "f493d512-5eb4-11ee-8c99-0242ac120002"
+DPI_ROUND_2_ID = "0059aad4-5eb5-11ee-8c99-0242ac120002"
+APPLICATION_BASE_PATH = ".".join([str(DPI_R2_BASE_PATH), str(1)])
+ASSESSMENT_BASE_PATH = ".".join([str(DPI_R2_BASE_PATH), str(2)])
+DPI_R2_OPENS_DATE = datetime(
+    2023, 10, 19, 10, 0, 0, tzinfo=timezone.utc
+)  # 2023-10-17 10:00:00
+DPI_R2_DEADLINE_DATE = datetime(
+    2023, 12, 1, 11, 59, 0, tzinfo=timezone.utc
+)  # 2023-11-1 11:59:00
+DPI_R2_ASSESSMENT_DEADLINE_DATE = datetime(
+    2024, 1, 31, 12, 0, 0, tzinfo=timezone.utc
+)  # 2023-01-31 12:00:00
```

Comment:
Nit, but maybe instead of having the comment with the readable string, we just make it readable in the code itself:

```python
from datetime import datetime, timezone

def create_datetime(date_str):
    return datetime.strptime(date_str, "%Y-%m-%d %H:%M:%S").replace(tzinfo=timezone.utc)

DPI_R2_OPENS_DATE = create_datetime("2023-10-19 10:00:00")
DPI_R2_DEADLINE_DATE = create_datetime("2023-12-01 11:59:00")
DPI_R2_ASSESSMENT_DEADLINE_DATE = create_datetime("2024-01-31 12:00:00")
``` 

(then all other funds would use this)